### PR TITLE
MNGSITE-437 - Intellij IDEA code style setting file is outdated

### PR DESCRIPTION
**Pull request by sparsick**
This setting file was created by importing checkstyle settings in idea. Then I compared it with @bmarwell self-created Intellji setting file for Maven projects and add his XML code settings.

### CODE REVIEW

**Comment by mthmulders** [comment]
I don't know what the current style guidelines are, so I cannot really review. But I'm definitely +1 for maintaining it in configuration files so we can all easily conform to them.

File: content/resources/developers/maven-idea-codestyle.xml [hunk 1]
```diff
@@ -216,17 +79,34 @@ under the License.
...
     <option name="WHILE_BRACE_FORCE" value="3" />
     <option name="FOR_BRACE_FORCE" value="3" />
     <option name="PARAMETER_ANNOTATION_WRAP" value="1" />
-    <option name="VARIABLE_ANNOTATION_WRAP" value="2" />
+    <option name="VARIABLE_ANNOTATION_WRAP" value="1" />
     <option name="SPACE_WITHIN_ANNOTATION_PARENTHESES" value="true" />
-    <option name="ENUM_CONSTANTS_WRAP" value="2" />
+    <option name="ENUM_CONSTANTS_WRAP" value="1" />
+    <indentOptions>
+      <option name="INDENT_SIZE" value="2" />
```
Comment (slawekjaranowski) [inline content/resources/developers/maven-idea-codestyle.xml:107]: indent size for java should be 4

File: content/resources/developers/maven-idea-codestyle.xml [hunk 2]
```diff
@@ -216,17 +79,34 @@ under the License.
...
+      <option name="CONTINUATION_INDENT_SIZE" value="4" />
+      <option name="TAB_SIZE" value="2" />
```
Comment (slawekjaranowski) [inline content/resources/developers/maven-idea-codestyle.xml:109]: tab size for java should be 4

File: content/resources/developers/maven-idea-codestyle.xml [hunk 3]
```diff
@@ -216,17 +79,34 @@ under the License.
...
     <option name="FOR_BRACE_FORCE" value="3" />
     <option name="PARAMETER_ANNOTATION_WRAP" value="1" />
-    <option name="VARIABLE_ANNOTATION_WRAP" value="2" />
+    <option name="VARIABLE_ANNOTATION_WRAP" value="1" />
     <option name="SPACE_WITHIN_ANNOTATION_PARENTHESES" value="true" />
-    <option name="ENUM_CONSTANTS_WRAP" value="2" />
+    <option name="ENUM_CONSTANTS_WRAP" value="1" />
+    <indentOptions>
+      <option name="INDENT_SIZE" value="2" />
+      <option name="CONTINUATION_INDENT_SIZE" value="4" />
```
Comment (slawekjaranowski) [inline content/resources/developers/maven-idea-codestyle.xml:108]: should be 8

File: content/resources/developers/maven-idea-codestyle.xml [hunk 4]
```diff
@@ -16,186 +16,49 @@ KIND, either express or implied.  See the License for the
...
-  <option name="ASSIGNMENT_WRAP" value="1" />
-  <option name="WRAP_COMMENTS" value="true" />
-  <option name="IF_BRACE_FORCE" value="3" />
-  <option name="DOWHILE_BRACE_FORCE" value="3" />
-  <option name="WHILE_BRACE_FORCE" value="3" />
-  <option name="FOR_BRACE_FORCE" value="3" />
-  <option name="SPACE_WITHIN_ANNOTATION_PARENTHESES" value="true" />
-  <option name="ENUM_CONSTANTS_WRAP" value="2" />
   <JavaCodeStyleSettings>
     <option name="ANNOTATION_PARAMETER_WRAP" value="2" />
```
Comment (slawekjaranowski) [inline content/resources/developers/maven-idea-codestyle.xml:26]: Maybe  1 - wrap if long

**Review by slawekjaranowski — CHANGES_REQUESTED** [review]

**Comment by sparsick** [comment]
@slawekjaranowski Thanks for reviewing

File: content/resources/developers/maven-idea-codestyle.xml [hunk 5]
```diff
@@ -16,186 +16,49 @@ KIND, either express or implied.  See the License for the
 specific language governing permissions and limitations
 under the License.
 -->
-<code_scheme name="maven">
-  <option name="JAVA_INDENT_OPTIONS">
-    <value>
-      <option name="INDENT_SIZE" value="4" />
-      <option name="CONTINUATION_INDENT_SIZE" value="4" />
-      <option name="TAB_SIZE" value="4" />
-      <option name="USE_TAB_CHARACTER" value="false" />
```
Comment (elharo) [inline content/resources/developers/maven-idea-codestyle.xml:25]: It's not clear to me why so much got deleted. Is this somehow rolled up into other changes in this file?

Comment (slawekjaranowski) [reply content/resources/developers/maven-idea-codestyle.xml:25]: New version of Intellij so new version of content.

Please choose some reference file which is correctly formatted and run code format in IntelliJ - nothing shouldn't change.

**Comment by jira-importer** [comment]
Resolve #1001

**Pull request closed without merging**